boards/arm/cy8cproto_062_4343w/cy8cproto_062_4343w.dts

```diff
@@ -26,6 +26,24 @@
 		zephyr,shell-uart = &uart5;
 		zephyr,bt_uart = &uart2;
 	};
+
+	/* Wifi configuration */
+	wifi-sdio {
```

Comment:
doesn't this device have an address, a bus...?

Reply:
added sdhc0 node.